src/components/button/Button.tsx

```diff
@@ -18,6 +18,7 @@ export interface IButtonProps {
   htmlType?: string;
   activated?: boolean;
   sticked?: boolean;
+  styleOverwrite?: any;
```

Comment:
Please review type

Reply:
@diegomart2000 I copied the type for the overwrite from the other components where the styleOverwrite was used. What should I replace it with?

Reply:
@steiningermona make it a string, css should be a string unless this causes troubles. If that's the case, we will look for a better type. Let me know.

Reply:
@diegomart2000 I tested it out with type string and it worked